### PR TITLE
Add warning for current version

### PR DESCRIPTION
**Pull request by willemvb**
Like @Gummibeer suggested https://twitter.com/devgummibeer/status/1415430630255570947

image

Some back-end love needed to finish this off:
- [x] version number for current and optionally latest version
- [x] package name in composer command
- [ ] currently displaying on every page, not sure if this needs extra conditions

### CODE REVIEW

**Comment by simonhamp** [comment]
Copy note: the version selector is on the left (not the right)

**Comment by Gummibeer** [comment]
Love it - and I think it should be on all pages as people enter from direct links, Google Searches and whatever else. So only displaying on the "Introduction" for example would only catch x%. I think that Laravel shows that warning also on all pages and not only the introduction one.

The only conditional I would add would be a sentence like
> You are on v3 but latest version is v4

In case you aren't on the latest version. To show that they could/should upgrade. I will also check if there are options for that command to reduce the output to the version or if there's another command to do so.

File: resources/views/front/pages/docs/show.blade.php [hunk 1]
```diff
@@ -15,6 +15,22 @@
...
+                <div class="flex-none h-6 w-6 text-orange fill-current">{{ svg('icons/fal-exclamation-circle') }}</div>
+                <div class="ml-4">
+                    <p>
+                    This is the documentation for <strong>v.{3}</strong> {but latest version is v4}.
+                    You can switch versions in the menu <span class="hidden md:inline">on the left</span><span class="hidden">/</span><span class="inline md:hidden">at the top</span>.
+                    Check your current version via following command:
+                    </p>
+                    <div class="mt-2">
+                        <code class="bg-blue-lightest bg-opacity-50 px-2 py-1">
+                            composer show {spatie/laravel-activitylog}
```
Comment (mallardduck) [inline resources/views/front/pages/docs/show.blade.php:28]: Should this be ready to copy-paste? Maybe we remove the braces to make that easier?

**Comment by riasvdv** [comment]
@willemvb Added the dynamic parts, can you double-check that everything looks as you intended?

**Comment by willemvb** [comment]
@riasvdv looking good! 
Added condition to only show this when there are multiple versions. 

Ready to ship!

**Comment by riasvdv** [comment]
Merged & deployed! 🚀

**Pull request merged**